src/mod_zone_difficulty_scripts.cpp

```diff
@@ -1321,10 +1355,10 @@ class mod_zone_difficulty_allcreaturescript : public AllCreatureScript
             {
                 if (creature->GetMaxHealth() == newHp)
                 {
-                    //if (sZoneDifficulty->IsDebugInfoEnabled)
-                    //{
-                    //    //LOG_INFO("module", "MOD-ZONE-DIFFICULTY: Modify creature hp for normal mode: {} to {}", baseHealth, baseHealth);
-                    //}
+                    if (sZoneDifficulty->IsDebugInfoEnabled)
+                    {
+                       //LOG_INFO("module", "MOD-ZONE-DIFFICULTY: Modify creature hp for normal mode: {} to {}", baseHealth, baseHealth);
```

Comment:
Let's uncomment this so it actually works in the debug mode.

Reply:
Haha yes, good idea.